packages/gamut/src/Badge/index.tsx

```diff
@@ -1,18 +1,58 @@
-import React from 'react';
+import {
+  states,
+  styledOptions,
+  system,
+  variant,
+} from '@codecademy/gamut-styles';
+import { StyleProps, variance } from '@codecademy/variance';
+import styled from '@emotion/styled';
 
-import { Box } from '../Box';
+const badgeVariants = variant({
+  base: {
+    borderRadius: '3px',
+    py: 4,
+    px: 12,
+    fontWeight: 400,
+    display: 'inline-block',
+    textAlign: 'center',
+    width: 'min-content',
+  },
+  defaultVariant: 'blue',
+  variants: {
+    blue: {
+      bg: 'blue',
+      textColor: 'white',
+    },
+    yellow: {
+      bg: 'feedback-warning',
+      textColor: 'black',
+    },
+    grey: {
+      bg: `navy-500`,
+      textColor: 'white',
+    },
+  },
+});
 
-export const Badge: React.FC = ({ children }) => (
-  <Box
-    bg="blue"
-    borderRadius="3px"
-    textColor="white"
-    display="inline-block"
-    fontSize={14}
-    mx={8}
-    py={4}
-    px={12}
-  >
-    {children}
-  </Box>
+const badgeStates = states({
+  round: {
+    borderRadius: '40px',
+  },
+});
+
+const badgeProps = variance.compose(
+  system.space,
+  system.layout,
+  system.typography
+);
+
+export interface BadgeProps
+  extends StyleProps<typeof badgeProps>,
+    StyleProps<typeof badgeVariants>,
+    StyleProps<typeof badgeStates> {}
+
+export const Badge = styled('div', styledOptions<'div'>())<BadgeProps>(
```

Comment:
`styledOptions` is by default typed to `div` so we don't need to call it with the generic arg in this case :)

```suggestion
export const Badge = styled('div', styledOptions)<BadgeProps>(
```